pkgs/development/libraries/libressl/default.nix

```diff
@@ -39,7 +39,7 @@ let
     meta = with lib; {
       description = "Free TLS/SSL implementation";
       homepage    = "https://www.libressl.org";
-      license = with licenses; [ publicDomain bsdOriginal bsd0 bsd3 gpl3 isc ];
+      license = with licenses; [ publicDomain bsdOriginal bsd0 bsd3 gpl3 isc openssl ];
```

Comment:
Where are `bsdOriginal bsd0 bsd3 gpl3` coming from?

Reply:
Good question. I don't see these mentioned anywhere in the license so I think they should be removed.

Reply:
I couldn't find them either.

Reply:
Actually, when you browse through the `libressl` source code you see that every source file is prefixed with its license. I've seen at least GPLv3 and BSD-3 but I'm not surprised the others are in there as well.